pkg/services/file.go

```diff
@@ -0,0 +1,69 @@
+package services
```

Comment:
I think It is good using s3.go to file name.

Reply:
Thank you! I updated